addons/point_of_sale/models/pos_bill.py

```diff
@@ -0,0 +1,16 @@
+from odoo import api, fields, models, _
+
+
+class Bill(models.Model):
+    _name = "pos.bill"
+    _order = "value"
+    _description = "Coins/Bills"
+
```

Comment:
Shouldn't we add a currency_id on the pos.bill? like that when we are creating a pos config, we'll set all the bills in the currency of the POS in the pos config.

In demo, you could let those demo with the currency "euro", and maybe add the demo for "dollar"

Reply:
@trj-odoo je vais faire une tache pour les currency, on fera ca dans la version suivante

Reply:
Faut parler en English ici pretty please